File: src/com/eagle/programmar/Go/Expressions/Go_StarExpression.java
// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Apr 1, 2024

package com.eagle.programmar.Go.Expressions;

import com.eagle.programmar.Go.Go_Expression;
import com.eagle.tokens.PrimaryOperator;
import com.eagle.tokens.punctuation.PunctuationStar;

public class Go_StarExpression extends PrimaryOperator
{
	public @S(10) PunctuationStar star;
	public @S(20) Go_Expression expression;
}

File: csrc/com/eagle/programmar/Go/Expressions/Go_StarExpression.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Apr 1, 2024

namespace com.eagle.programmar.Go.Expressions
{
	using Go_Expression = com.eagle.programmar.Go.Go_Expression;
	using PrimaryOperator = com.eagle.tokens.PrimaryOperator;
	using PunctuationStar = com.eagle.tokens.punctuation.PunctuationStar;

	public class Go_StarExpression : PrimaryOperator
	{
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(10) com.eagle.tokens.punctuation.PunctuationStar star;
		public PunctuationStar star;
// JAVA TO C# CONVERTER TASK: Most Java annotations will not have direct .NET equivalent attributes:
// ORIGINAL LINE: public @S(20) com.eagle.programmar.Go.Go_Expression expression;
		public Go_Expression expression;
	}

}
